op-geth.yaml

```diff
@@ -37,6 +37,22 @@ pipeline:
       output: geth
 
 subpackages:
+  - name: "${{package.name}}-compat"
+    description: "Compatibility package to place binaries in the same location used by the upstream image"
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/usr/local/bin
+          ln -sf /usr/bin/geth ${{targets.subpkgdir}}/usr/local/bin/geth
```

Comment:
is something not working as expected on images side? 

upstream uses the following entrypoint.
```bash
$ crane config us-docker.pkg.dev/oplabs-tools-artifacts/images/op-geth | jq .config.Entrypoint
[
  "geth"
]
```

just by looking at it, my assumption is that it should work but I must be missing some details that came up as part of testing. It would be great if you can provide those. thank you!!

Reply:
Hi @kranurag7, The existing package works perfectly and does not block any testing. However, for sanity and compatibility reasons, Wojciech pointed out that upstream places the geth binary in [/usr/local/bin](https://github.com/ethereum-optimism/op-geth/blob/optimism/Dockerfile#L23), while our image places it in /usr/bin. To ensure compatibility for users relying on `/usr/local/bin/geth,` I added a compact package that creates a symlink from `/usr/bin/geth` to `/usr/local/bin/geth`. This helps maintain consistency with the upstream behavior.

Reply:
thank you providing the context, this looks good to me.